### PR TITLE
Roman_Numeral

### PR DESCRIPTION
**Pull request by Marwa-Sadat**
1. *Objective:* An app was created to convert numbers to Roman numerals and vice versa. ⚙🔢➡🏛
2. *Inputs:* Fields were provided for entering numbers and Roman numerals. ✏
3. *Conversion:* Logic was implemented to accurately handle both conversions. 🔄
4. *Validation:* Users were alerted for invalid inputs or formats. 🚫❗
5. *Styling:* A responsive and visually appealing user interface was designed. 🎨📱
6. *Deployment:* The app was hosted and ensured to be accessible online. 🌐
Screenshot 2024-06-29 115526

### CODE REVIEW

**Review by zainabbehzad — CHANGES_REQUESTED** [review]
Hi @Marwa-Sadat
Excellent work You have done a great job implementing the project requirements.

✔Great design implemented.
✔Created a professional Repo.
✔Correct use of Gitflow.
✔Commit messages are meaningful.
✔Setup the linters correctly.
✔No linters errors.
✔You have followed and completed the Palindrome Checker and all requirements.
✔Your codes passes all FreeCodeCamp requiremenrs.
✔Created the pull request (From your  Roman-Numeral branch to the development branch you created a pull request and submitted the pull request link for review).
✔Good PR description.
✔Your repo has a professional README.md.
✔Your project design is responsive.
🧩Your code is clean.
  On index.html
* Your body spacing is not aligned correctly:
* All child elements should be two spaces ahead of their parent elements.

In index.html you should not add CSS codes as you have external CSS file. And added this codes in your CSS file.
Also you should clear your comments when you finished your project (styles.css).

File: index.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+         cursor: pointer;
+         }
+         #convert-btn:hover {
+         background-color: #0056b3;
+         }
+         #output {
+         margin-top: 20px;
+         font-size: 18px;
+         }
+      </style>
```
Comment (zainabbehzad) [inline index.html]: You should delete all these CSS codes from HTML file  instead link your CSS file in HTML file.
CSS link: <link rel="stylesheet" href="styles.css">

**Comment by Marwa-Sadat** [comment]
Hi @zainabbehzad   I have done your feedback, please check my project .

**Review by zainabbehzad — APPROVED** [review]
Hi @Marwa-Sadat
Excellent work You have done a great job implementing the project requirements.

✔Great design implemented.
✔Created a professional Repo.
✔Correct use of Gitflow.
✔Commit messages are meaningful.
✔Setup the linters correctly.
✔No linters errors.
✔You have followed and completed the Palindrome Checker and all requirements.
✔Your codes passes all FreeCodeCamp requiremenrs.
✔Created the pull request (From your  Roman-Numeral branch to the development branch you created a pull request and submitted the pull request link for review).
✔Good PR description.
✔Your repo has a professional README.md.
✔Your project design is responsive.
✔Your code is clean.
  
Suggestion:
Just a little more work on your README.md

**Comment by Marwa-Sadat** [comment]
Thank you
***@***.*** 😊 *

**Comment by Marwa-Sadat** [comment]
@ahzamir I hope you are well, I have done all the feedback, please check my project.

**Review by ahzamir — APPROVED** [review]
### Hi @Marwa-Sadat 🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text with a screenshot.

- Well Done great design Implemented. 🎯🎉👍
image

# Status: Approved ✅
There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳

WinkThumbsUpGIF

 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Roman Numeral Converter project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your code is clean
✔️Your repo has a professional README.md file
✔️ Your partner has reviewed your repo
✔️Your design is responsive

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me @ahzamir in your question so I can receive the notification.**

**Comment by Marwa-Sadat** [comment]
@ahzamir thank you so much sir 😊🙂.....

**Pull request merged**